app/components/navigation/google-login-btn.tsx

```diff
@@ -0,0 +1,55 @@
+import Script from "next/script";
+import { useEffect, useState, useRef } from "react";
+
+interface GoogleLogin {
+    credential: string,
+    select_by: string
+}
+
+export default function GoogleLoginButton(){
+    const [scriptLoaded, setScriptLoaded] = useState(false);
+    const handleScriptLoad = () => {
+        setScriptLoaded(true);
+    };
+
+    const googleLoginRef = useRef(null);
+
+    useEffect(()=> {
+        if(scriptLoaded){
+            const google = window.google;
+            google.accounts.id.initialize({
+                client_id: "708154320268-432vdrg2g0h1652frag5vbu8r8qi4ers.apps.googleusercontent.com",
```

Comment:
Shouldn't this come from somewhere else and not be checked in 😅

Reply:
I thought so as well, but unsure the best place to put it

Reply:
Do we have documentation that we are working from to set this up?

Reply:
We just going off of [this](https://developers.google.com/identity/sign-in/web/sign-in)

Reply:
This will be addressed in this issue: https://github.com/jjm3x3/AmazingRaceFantasy/issues/291

Reply:
I also just created #293 to at least update this value to work with the account created in #277